mobile/app/src/main/AndroidManifest.xml

```diff
@@ -43,29 +43,25 @@
             <!-- Handles the implicit intent to VIEW the intel.ingress.com/ URI -->
             <intent-filter>
                 <action android:name="android.intent.action.VIEW"/>
-                <action android:name="android.nfc.action.NDEF_DISCOVERED"/>
 
                 <category android:name="android.intent.category.DEFAULT"/>
                 <category android:name="android.intent.category.BROWSABLE"/>
 
-                <data android:scheme="http"/>
-                <data android:scheme="https"/>
-                <data android:host="intel.ingress.com"/>
+                <data android:scheme="http" android:host="intel.ingress.com"/>
```

Comment:
As far as I remember, it follows from the documentation that the previous version also works